refact: create `vello_example_scenes` crate with common examples (#1236)
linebender/vello: sparse_strips/vello_example_scenes/README.md

# vello_example_scenes

> A collection of scenes used in Vello Hybrid examples.

Reviewer: ```suggestion
A collection of scenes used for Vello examples.
```

Nit: Do not make this specific to hybrid, as examples are also used for CPU.
